### ISSUE TITLE
SoftHSMv2 can't login as Security Officer

### ISSUE THREAD

**Issue by iTestAndroid** [PeculiarVentures/graphene#77]
I try to login as SO (Security Officer) with SoftHSMv2 and I always get:
CKR_SESSION_READ_ONLY_EXISTS

No other session is open.

**Comment by microshine** [comment]
[Here](https://github.com/PeculiarVentures/graphene/blob/master/test/session.js#L68-L82) is working test for this case

**Comment by iTestAndroid** [comment]
Yes, but I already know the SO password and I try to login with SO with `graphene.UserType.SO` argument. I had to manually go and fix/patch SoftHSMv2 code to just ignore this error.
